### PR TITLE
Fix migration publishing

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
## Issue
Published package was missing lib and lib-commonjs code

## Changes
- Updated ts.config to build to dist folder like a v8 component

### CODE REVIEW

File: change/@fluentui-react-migration-v8-v9-3cbbe105-994c-407f-9441-2c2ce867d5f0.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Updated to tsconfig to publish to dist",
```
Comment (khmakoto) [inline change/@fluentui-react-migration-v8-v9-3cbbe105-994c-407f-9441-2c2ce867d5f0.json:3]:
```suggestion
  "comment": "fix: Updated to tsconfig to publish to dist.",
```

**Review by khmakoto — APPROVED** [review]

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/fe895d6bb4f9df0304f63f7e77acafef4b912925'>fe895d6bb4f9df0304f63f7e77acafef4b912925</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by TristanWatanabe** [comment]
I think you need to update the `api.extractor.json` file to use the old setup as well

```
{
  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
  "extends": "@fluentui/scripts/api-extractor/api-extractor.common.v-next.json",
  "mainEntryPointFilePath": "<projectFolder>/dist/index.d.ts"
}
```

File: packages/react-migration-v8-v9/config/api-extractor.json [hunk 2]
```diff
@@ -1,10 +1,5 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
   "extends": "@fluentui/scripts/api-extractor/api-extractor.common.v-next.json",
-  "dtsRollup": {
-    "enabled": true,
-    "untrimmedFilePath": "",
-    "publicTrimmedFilePath": "<projectFolder>/dist/index.d.ts"
-  },
-  "mainEntryPointFilePath": "<projectFolder>/../../dist/out-tsc/types/packages/<unscopedPackageName>/src/index.d.ts"
+  "mainEntryPointFilePath": "<projectFolder>/dist/index.d.ts"
```
Comment (TristanWatanabe) [inline packages/react-migration-v8-v9/config/api-extractor.json:4]:
```suggestion
  "mainEntryPointFilePath": "<projectFolder>/dist/types/index.d.ts"
```

**Review by TristanWatanabe — APPROVED** [review]

**Pull request merged**